_studio/mfx_lib/encode/CMakeLists.txt

```diff
@@ -1,26 +1,33 @@
 set(MFX_ORIG_LDFLAGS "${MFX_LDFLAGS}" )
 
-list( APPEND vdirs
-  mjpeg
-  )
-list( APPEND cdirs
-  jpeg_enc color_space_converter
-  )
 mfx_include_dirs( )
-foreach( dir ${vdirs} )
-  include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/${dir}/include )
-endforeach()
 
-foreach( dir ${cdirs} )
-  include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/../../shared/umc/codec/${dir}/include )
-endforeach( )
+include_directories( ${CMAKE_CURRENT_SOURCE_DIR}/mjpeg/include )
 
 set( defs "${API_FLAGS} ${WARNING_FLAGS}" )
```

Comment:
@Alexandr-Konovalov : I know that's not your change, but can you, please, remove this line? Really, we don't have these variables on github. Please, just add:
`set( defs "" )`

Reply:
Done.